src/app/Table.elm

```diff
@@ -0,0 +1,29 @@
+module Table exposing (view, init, Msg)
```

Comment:
Can you point at some code where this is working in a way that decouples the pieces? "steering the community away" is ok, but there's no concrete advice as to where the steering is ending up.